Exception in thread "main" java.lang.NullPointerException
	at Solution.twoSum(Test.java:4)
	at Test.main(Test.java:13)
